### DISCUSSION TITLE
Folder naming rule when pull request

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#7] [General]
@hi-hyein  @yoonucho @Joseph-Cha

제가 동훈님과 얘기하다가 pull request 할 때 어느 경로에다가 할지 안정한거 같아서 적어 봅니다.

지금 읽는 책이 <소프트웨어 스펙의 모든 것>인데 영어 제목을 AllOfSRS라고 해보면 좋을 것 같습니다.

그래서
년도/책제목/멤버이름/챕터이름.md
이런식으로 해보면 좋을 것 같고 그러면 아래와 같은 구조로 나오게 되겠죠.
/2021/AllOfSRS/<_yourname_>/chapter1.md

그리고 chapter1.md, chapter2.md, chapter3.dm의 파일을 따로 작성한 후에
pull request 한번에 파일 3개를 요청하는 식으로 하면 좋을 거 같습니다.

저의 경우는
/2021/AllOfSRS/jongfeel/chapter1.md
/2021/AllOfSRS/jongfeel/chapter2.md
/2021/AllOfSRS/jongfeel/chapter3.md

이런 식으로 될 겁니다.

---

SRS(Software Requirements Specification)는 위키를 확인해 보시면 됩니다.
https://en.wikipedia.org/wiki/Software_requirements_specification

**Comment by hi-hyein** [comment]
좋은 의견입니다! 이렇게 진행하면 좋을 것 같아요!
